server.js

```diff
@@ -232,7 +232,7 @@ app.get('/come-back', (req, res) => {
     const jsReferrer = document.referrer;
     document.body.innerHTML += '<p>header: <strong>${req.headers.referer || ''}</strong></p><p>js: <strong>' + jsReferrer + '</strong></p>';
     setTimeout(() => {
-        location.href = 'https://privacy-test-pages.glitch.me/privacy-protections/referrer-trimming/?run&header=${req.headers.referer || ''}&js=' + jsReferrer + '&testid=${req.query.testid || ''}';
+        location.href = 'https://privacy-test-pages.site/privacy-protections/referrer-trimming/?run&header=${req.headers.referer || ''}&js=' + jsReferrer + '&testid=${req.query.testid || ''}';
```

Comment:
This will basically redirect some of the tests happening on glitch.me to privacy-test-pages.site mid test. Might not be a big deal if we don't care about glitch.me anymore

Reply:
Closing - we agreed that we will decommission glitch.me